index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- Font Lato and Staatliches -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <!--Font Awesome-->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+  integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <!-- Stylesheets -->
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/mobile.css">
+  <title>World Summit Amsterdam Noord 2022</title>
+</head>
+<body id="homepage">
+  <!-- Header -->
+  <header class="menu-bar">
+    <!-- Main Top Bar -->
+    <div id="top-bar">
+      <button id="close" type="button"><img src="images/close.svg" alt="close mobile menu button"></button>
+      <button id="menu" type="button"><img src="images/mobile_menu.svg" alt="open mobile menu button"></button>
+      <ul class="top-bar-links">
+        <li class="top-bar-link"><a href="https://www.facebook.com"><i class="fa-brands fa-facebook-square"></i></a></li>
+        <li class="top-bar-link"><a href="https://www.twitter.com"><i class="fa-brands fa-twitter-square"></i></a></li>
+        <li class="top-bar-link"><a href="#">Spanish</a></li>
+        <li class="top-bar-link"><a href="#">My Page</a></li>
+        <li class="top-bar-link"><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <!-- Nav Bar -->
```

Comment:
- [ ] The layout of the bottom header changes when I move from the Home page to the About page. Please make sure it has same header. 
- [ ] When we are on About page there should be a Home Page button instead of Program.